README.md

```diff
@@ -98,4 +98,35 @@ $ ansible-playbook my-ansible-playbook.yml -i ansible-inventory.ini
 
 ## Module 4 stretch goal
 
-$ poetry add gunicorn
\ No newline at end of file
+$ poetry add gunicorn
+
+## Run in Docker
+
+#### Development Mode
+
+1. Build the development image
+   '''bash
+   docker build --target development --tag todo-app:dev .
+   '''
+2. Run the container (2 options)
+   2.1 Docker Command
+   '''bash
+   docker run --env-file .env --publish 5000:80 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app/todo_app todo-app:dev
+   '''
```

Comment:
Similarly here (and a couple more below)
````suggestion
   ```bash
   docker run --env-file .env --publish 5000:80 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app/todo_app todo-app:dev
   ```
````